ext/web/lib.rs

```diff
@@ -149,8 +149,25 @@ fn op_base64_decode(
   _state: &mut OpState,
   input: String,
   _: (),
-) -> Result<ZeroCopyBuf, AnyError> {
-  let mut input: &str = &input.replace(|c| char::is_ascii_whitespace(&c), "");
+) -> Result<String, AnyError> {
+  let mut ws = false;
+  let len = input.len();
+  let invalid = input.char_indices().any(|(i, c)| {
+    if char::is_ascii_whitespace(&c) {
+      ws = true;
+    } else if c == '=' {
+      if i != len - 1 && i != len - 2 {
```

Comment:
There is a bit of a possibility for false negative here: Even if there are '=' characters at the end it does not mean that they'll be removed below since that depends on whitespace removal etc.

As such, it would be more appropriate to track '=' markers at end with an enum: None, One, Two. Then compare to those when '=' is being removed and reset the enum appropriately. If reset does not happen, it means that the '=' at the end were invalid characters.